requirements.txt

```diff
@@ -1,4 +1,5 @@
-numpy~=1.23
+numpy>=1.26
+ml_dtypes>=0.4.0
```

Comment:
Do we need to define this? Is it a soft/hard dependency?

Reply:
Just freed `ml_dtypes`